bin/generate_commit

```diff
@@ -79,7 +79,16 @@ function _generate_schemas() {
 
     # Copy aliased BQ schemas into place
     alias_schemas "$ALIASES_PATH" .
+
     popd
+
+    # Copy all BQ schemas from schemas/ to appropriate paths under bq/
+    # TODO: Remove the exemption for xfocsp-error-report/xfocsp-error-report
+    # once ops logic has adapted to the bq/ directory structure.
+    find schemas/ -name '*.bq' | \
+        grep -v xfocsp-error-report/xfocsp-error-report | \
+        xargs copy_to_bq_folder
```

Comment:
You may want to do this before popping out of the schemas directory.

Reply:
This script is copying schemas from underneath `schemas/` into `bq/` which is a new top-level folder. It seems easier to reason about this operation from the parent directory under which both `schemas/` and `bq/` live rather than referencing `../bq` from within `schemas`.

Reply:
The function doesn't necessarily need to be run from the root of the docker application though. I would at least suggest referencing the `$mps_root` directory that's being passed in.

Reply:
I've pulled this line above the `popd`.